### DISCUSSION TITLE
Storing emails in postgres and storage

### DISCUSSION THREAD

**Discussion by rbkayz** [supabase/supabase#7205] [Questions]
@supabase team, could you please help with any tutorial on how could one store emails. I presume storing message metadata on postgres, and html in storage is a good solve. But is there a walkthrough on how you can set the same RLS policies on the storage buckets? Ty

**Comment by dshukertjr** [comment]
Hi @rbkayz,

just to clarify, you want to store actual email content on Supabase and not people's email address?

**Comment by rbkayz** [comment]
Yes Tyler, that's correct. We want to store the email metadata (senders address, receivers address, timestamp, message header, etc.), and the message body (along with any assets (like attachments)

**Reply by dshukertjr** [reply]
@rbkayz 
Thanks for the clarification! That sounds perfectly suitable! Just one question. You asked for some examples on row level security policies around storage, but who should be able to view the email contents in this case? If you could elaborate more on what the reason you are storing these information as well as more descriptions on what your app will do would help us guide you to the right direction!
